test/transcoding/service.pb.txt

```diff
@@ -481,6 +486,27 @@ types {
   }
   syntax: SYNTAX_PROTO3
 }
+types {
+  name: "endpoints.examples.bookstore.UnknownBook"
+  fields {
```

Comment:
FYI,  I manually deleted "unknown" field and "UknownMsg" from the service config.